src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -148,6 +152,11 @@ export default {
 		KvTextInput,
 	},
 	inject: ['apollo', 'cookieStore'],
+	apollo: {
```

Comment:
This should already be handled globally here: https://github.com/kiva/ui/blob/4085cf13460d40fc9771fd793128d222847b246a/src/util/experiment/experimentUtils.js#L343

Does this still work like expected with this `preFetch` removed?